spec.html

```diff
@@ -35362,6 +35463,157 @@ <h1>Static Semantics: CharacterValue ( ): a non-negative integer</h1>
           1. Let _ch_ be the code point matched by |IdentityEscape|.
           1. Return the numeric value of _ch_.
         </emu-alg>
+        <emu-grammar>ClassSetCharacter :: SourceCharacter but not ClassSetSyntaxCharacter</emu-grammar>
+        <emu-alg>
+          1. Let _ch_ be the code point matched by |SourceCharacter|.
+          1. Return the numeric value of _ch_.
+        </emu-alg>
+        <emu-grammar>ClassSetCharacter :: `\` ClassSetReservedPunctuator</emu-grammar>
+        <emu-alg>
+          1. Let _ch_ be the code point matched by |ClassSetReservedPunctuator|.
+          1. Return the numeric value of _ch_.
+        </emu-alg>
+        <emu-grammar>ClassSetCharacter :: `\b`</emu-grammar>
+        <emu-alg>
+          1. Return the numeric value of U+0008 (BACKSPACE).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-static-semantics-maycontainstrings" type="sdo">
+        <h1>Static Semantics: MayContainStrings ( ): a Boolean</h1>
+        <dl class="header">
+        </dl>
+        <emu-grammar>
+          CharacterClassEscape ::
+            `d`
+            `D`
+            `s`
+            `S`
+            `w`
+            `W`
+            `P{` UnicodePropertyValueExpression `}`
+
+          UnicodePropertyValueExpression ::
+            UnicodePropertyName `=` UnicodePropertyValue
+
+          CharacterClass ::
+            `[^` ClassContents `]`
+
```

Comment:
I think it's unnecessary to define `MayContainStrings` on this production.

Reply:
How so?

Reply:
Additionally, for chain productions (productions that includes only one non-terminal, `ClassContents` in this case), syntax-directed operations have a default definition which just returns the result of applying the SDO to the non-terminal.

So even if this was needed, it would already return false by default, right?

Ref: the "Unless explicitly specified otherwise" paragraph of https://tc39.es/ecma262/#sec-algorithm-conventions-syntax-directed-operations

Reply:
https://github.com/mathiasbynens/ecma262/pull/24 merged. Thanks!